src/main/java/apoc/schema/Schemas.java

```diff
@@ -269,15 +269,50 @@ private Boolean constraintsExistsForRelationship(String type, List<String> prope
      *
      * @return
      */
-    private Stream<IndexConstraintNodeInfo> indexesAndConstraintsForNode() throws IndexNotFoundKernelException {
+    private Stream<IndexConstraintNodeInfo> indexesAndConstraintsForNode(Map<String,Object> config) throws IndexNotFoundKernelException {
+
+        SchemaConfig schemaConfig = new SchemaConfig(config);
+        Collection<String> includeLabels = schemaConfig.getLabels();
+        Collection<String> excludeLabels = schemaConfig.getExcludeLabels();
+
         try ( Statement ignore = tx.acquireStatement() ) {
             TokenRead tokenRead = tx.tokenRead();
             TokenNameLookup tokens = new SilentTokenNameLookup(tokenRead);
 
             SchemaRead schemaRead = tx.schemaRead();
-            Iterable<IndexReference> indexesIterator = () -> schemaRead.indexesGetAll();
+            Iterable<IndexReference> indexesIterator;
+            Iterable<ConstraintDescriptor> constraintsIterator;
+
+            if ((includeLabels == null || includeLabels.isEmpty()) ||
```

Comment:
Cosa ci aspettiamo da questo if?

Reply:
Praticamente gestisce i casi in cui o labels non è settato, oppure sono settati sia labels che excludeLabels. Nel primo caso vengono estratti tutti gli indici e constraint, nel secondo tutti tranne quelli in exlude

Reply:
Sulla base della validazione del commento sopra semplificare l'if